plugins/source/aws/resources/services/cloudwatchlogs/resource_policies.go

```diff
@@ -0,0 +1,53 @@
+// Code generated by codegen; DO NOT EDIT.
+
+package cloudwatchlogs
+
+import (
+	"github.com/cloudquery/cloudquery/plugins/source/aws/client"
+	"github.com/cloudquery/plugin-sdk/schema"
+)
+
+func ResourcePolicies() *schema.Table {
+	return &schema.Table{
+		Name:        "aws_cloudwatchlogs_resource_policies",
+		Description: `https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_ResourcePolicy.html`,
+		Resolver:    fetchCloudwatchlogsResourcePolicies,
+		Multiplex:   client.ServiceAccountRegionMultiplexer("logs"),
+		Columns: []schema.Column{
+			{
+				Name:     "account_id",
+				Type:     schema.TypeString,
+				Resolver: client.ResolveAWSAccount,
+				CreationOptions: schema.ColumnCreationOptions{
+					PrimaryKey: true,
+				},
+			},
+			{
+				Name:     "region",
+				Type:     schema.TypeString,
+				Resolver: client.ResolveAWSRegion,
+				CreationOptions: schema.ColumnCreationOptions{
+					PrimaryKey: true,
+				},
+			},
+			{
+				Name:     "last_updated_time",
+				Type:     schema.TypeInt,
+				Resolver: schema.PathResolver("LastUpdatedTime"),
+			},
+			{
+				Name:     "policy_document",
+				Type:     schema.TypeString,
+				Resolver: schema.PathResolver("PolicyDocument"),
```

Comment:
`PolicyDocument`s are sometimes `schema.TypeJSON` and not `schema.TypeString` (we both have it in different resources, not sure about the reason of the inconsistency) did/can you check if using TypeJSON would be better here or not?